qmi/instruments/tenma/psu_72.py

```diff
@@ -0,0 +1,490 @@
+    MAX_CURRENT = 3
+
+    @rpc_method
+    def get_status(self) -> Dict[str, Any]:
+        """Get the power supply status as a dictionary of status values.
+
+        Dictionary composition is:
+            Ch1Mode: "C.V" | "C.C"
+            Ch2Mode: "C.V" | "C.C"
+            Tracking: xx, where xx is byte
+                * 00 = Independent
+                * 01 = Tracking series
+                * 10 = Tracking parallel
+            OutputEnabled: True | False
+        
+        Returns: 
+            Dictionary of status values.
+        """
+        self._send("STATUS?")
+        statusByte = ord(self._transport.read(1, 1))  # Read response byte
+        ch1mode = (statusByte & 0x01)
+        ch2mode = (statusByte & 0x02)
+        tracking = (statusByte & 0x0C) >> 2
+        output_enabled = (statusByte & 0x40)
+
+        if tracking == 0:
+            tracking = "Independent"
+        elif tracking == 1:
+            tracking = "Tracking Series"
+        elif tracking == 2:
+            tracking = "Tracking Parallel"
+        else:
+            tracking = "Unknown"
+        return {
+            "Ch1Mode": "C.V" if ch1mode else "C.C",
+            "Ch2Mode": "C.V" if ch2mode else "C.C",
+            "Tracking": tracking,
+            "OutputEnabled": bool(output_enabled),
+        }
+
+    @rpc_method
+    def enable_output(self, output: bool) -> None:
+        command = f"OUT{int(output)}"
+        self._enable_output(output, command)
+
+
+class Tenma72_2535(Tenma72_2550):
+    MAX_VOLTAGE = 30
+    MAX_CURRENT = 3
+
+
+class Tenma72_2540(Tenma72_2550):
+    MAX_VOLTAGE = 30
+    MAX_CURRENT = 5
+
+
+class Tenma72_2545(Tenma72_2550):
+    MAX_VOLTAGE = 60
+    MAX_CURRENT = 2
+
+
+class Tenma72_2925(Tenma72_2550):
+    MAX_VOLTAGE = 30
+    MAX_CURRENT = 5
+
+
+class Tenma72_2930(Tenma72_2550):
+    MAX_VOLTAGE = 30
+    MAX_CURRENT = 10
+
+
+class Tenma72_2935(Tenma72_2550):
+    MAX_VOLTAGE = 60
+    MAX_CURRENT = 5
+
+
+class Tenma72_2940(Tenma72_2550):
+    DEFAULT_BAUDRATE = 9600
+    MAX_VOLTAGE = 60
+    MAX_CURRENT = 5
+
+
+class Tenma72_10480(Tenma72_2550):
+    MAX_VOLTAGE = 30
+    MAX_CURRENT = 3
+
+
+class Tenma72_13350(Tenma72_Base):
+    """Instrument driver for the Tenma 72-13350. The driver is tested with this model, but manual is
+    also for the model 72-13360.
+
+    This driver can be used with (USB-to-)serial and UDP communications.
+
+    Note that this model has extra functionalities related to the IP LAN Communication Protocol.
+    Also note that this model does not work with channel number input, like 2550 model-based PSUs.
+    """
+    MAX_VOLTAGE = 30
+    MAX_CURRENT = 30
+    DEFAULT_BAUDRATE = 115200
+
+    def __init__(self, context: QMI_Context, name: str, transport: str) -> None:
+        super().__init__(context, name, transport)
+        self.serial_eol = "\n"
+
+    def _send(self, cmd: str) -> None:
+        cmd = cmd + self.serial_eol
+        return super(Tenma72_13350, self)._send(cmd)
```

Comment:
why do you need arguments for this `super`? Same for other instances

Reply:
Well, my IDE filled it in automatically and I left it. But, in Python 3, the `super()` class knows how to fill these values in automagically, so those arguments are not actually needed. Removing... The result is exactly same.

Reply:
I realised with the other comment that I skimmed over the classname. This is what causes the recursions, because you are calling the `_send` method of `Tenma72_13350` inside its own `_send` method. I'm surprised `get_status` did not complain with the recursion error

Reply:
addressed in other comment